decks/Makefile

```diff
@@ -127,29 +129,34 @@ ifeq ($(GC),YES)
 	@echo "***************************************************************"
 	@echo "Compiling GEOS-Chem"
 	@echo "***************************************************************"
-	cd $(MODEL_DIR)/geos-chem/build && rm -rf * && cmake ../ -DINSTALLCOPY=.. && make -j install
-	#cd $(MODEL_DIR)/geos-chem/build && make install
-	cp $(MODEL_DIR)/geos-chem/build/mod/*.mod $(MODEL_DIR)/mod/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/mod/*.mod $(MODEL_DIR)/mod/
-	cp $(MODEL_DIR)/geos-chem/build/src/Cloud-J/mod/*.mod $(MODEL_DIR)/mod/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Core/libHCO.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Extensions/libHCOX.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Shared/Headers/libHeadersHco.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Shared/GeosUtil/libJulDayHco.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Shared/GeosUtil/libGeosUtilHco.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Shared/NcdfUtil/libNcdfUtilHco.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/HEMCO/src/Interfaces/Shared/libHCOI_Shared.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/ISORROPIA/libIsorropia.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/Headers/libHeaders.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/ObsPack/libObsPack.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/History/libHistory.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/KPP/fullchem/libKPP_FirstPass.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/KPP/fullchem/libKPP.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/GeosCore/libGeosCore.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/GeosUtil/libGeosUtil.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/GeosUtil/libJulDay.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/GEOS-Chem/NcdfUtil/libNcdfUtil.a $(MODEL_DIR)/geos-chem/lib/
-	cp $(MODEL_DIR)/geos-chem/build/src/Cloud-J/src/Core/libCloudJ_Core.a $(MODEL_DIR)/geos-chem/lib/
+	mkdir -p $(GC_BUILD_DIR)
+	mkdir -p $(GC_LIB_DIR)
+	cd $(GC_BUILD_DIR) && cmake .. -DINSTALLCOPY=.. \
+		-DOMP=$(MP) \
```

Comment:
`MP` is GISS' flag determining whether to run with OpenMP (off by default). This line ensures that the same configuration is used within GEOS-Chem, too.